src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -628,18 +628,11 @@ protected DynamoModel(IStartConfiguration config)
 
             UpdateManager = config.UpdateManager ?? new DefaultUpdateManager(null);
 
-            var hostUpdateManager = config.UpdateManager;
-
-            if (hostUpdateManager != null)
```

Comment:
UpdateManager will exist either way from above code so we can just check it and set the value directly, no need to consider else case